src/fs/copy.js

```diff
@@ -1,5 +1,17 @@
+import fs from "fs";
+
 const copy = async () => {
-    // Write your code here 
+  const dirPath = './files'
+  const newDirPath = './files_copy'
```

Comment:
Good idea to move filenames to separate module, e.g. 'constants' and then import it.